src/tests/multihost/alltests/test_automount.py

```diff
@@ -17,6 +17,14 @@
 JOURNALCTL_CMD = "journalctl -x -n 50 --no-pager"
 
 
+def restart_autofs(multihost):
+    cmd = 1
+    while cmd !=0:
```

Comment:
The main object of this function is to restart autofs. I dont know how many try it will take last successful try was 12 times . If i limit it to 10, it might not work , let me increase it to 20 times